python3 variable_error.py